listener/pkg/event/watcher_event_test.go

```diff
@@ -54,20 +54,20 @@ func TestUnmarshalSKREvent(t *testing.T) {
 		t.Run(testCase.name, func(t *testing.T) {
 			t.Parallel()
 			// GIVEN
-			url := fmt.Sprintf("%s%s", hostname, testCase.urlPath)
+			url := fmt.Sprintf("%s%s", hostname, testCase.urlPath) //nolint:govet
```

Comment:
It would be better if we fix the linting issue instead of ignoring the error here.
I would suggest removing the `//nolint:govet` comments and refactoring the loop variables on line 53 and use an index like the following:
```golang
for _, testCase := range testCases { //nolint:paralleltest
```
```golang
for idx := range testCases { //nolint:paralleltest
    testCase := testCases[idx]
```

Reply:
Good point. I misunderstood the error here and tthought that it couldn't be solved

Reply:
@khlifi411 This is in fact not possible due to the nature of parallel testing. The test will fail randomly if we assign the testCase variable before the t.Parallel() call. And we have to give the test a name so we need an object of testCase to call t.Run(). So either idx or testCase must be accessed in the function.

Reply:
@khlifi411 @LeelaChacha let me double check this again. I will do some research and come back to you

Reply:
As agreed offline, had a deeper look and raised this: https://github.com/kyma-project/runtime-watcher/pull/98